scripts/generate.js

```diff
@@ -72,7 +72,7 @@ async function getJsonSchemaToTypescript() {
       `
         set -eu
         cd ./node_modules/json-schema-to-typescript
-        rm -rf dist
+        rm -rf lib
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏This should stay `dist`

```suggestion
        rm -rf dist
```